packages/e2e-playwright/tests/dropin/sessions/dropin.sessions.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { test, expect } from '../../../pages/dropin/dropin.fixture';
+import { REGULAR_TEST_CARD, TEST_CVC_VALUE, TEST_DATE_VALUE } from '../../utils/constants';
+import { typeIntoSecuredField } from '../../../models/dropinModelUtils/utils';
+
+import LANG from '../../../../lib/src/language/locales/en-US.json';
+
+const CARD_IFRAME_TITLE = LANG['creditCard.encryptedCardNumber.aria.iframeTitle'];
+const EXPIRY_DATE_IFRAME_TITLE = LANG['creditCard.encryptedExpiryDate.aria.iframeTitle'];
+const CVC_IFRAME_TITLE = LANG['creditCard.encryptedSecurityCode.aria.iframeTitle'];
+
+const CARD_IFRAME_LABEL = LANG['creditCard.cardNumber.label'];
+const EXPIRY_DATE_IFRAME_LABEL = LANG['creditCard.expiryDate.label'];
+const CVC_IFRAME_LABEL = LANG['creditCard.securityCode.label'];
+
+test.describe('Dropin Sessions flow', () => {
+    test('#1 Should succeed in making a payment', async ({ dropinSessions_regular }) => {
+        const { dropin, page } = dropinSessions_regular;
+
+        await dropin.isComponentVisible();
+
+        const creditCard = dropin.getPaymentMethodItem('Cards');
+        await creditCard.scrollIntoViewIfNeeded();
+
+        await page.waitForTimeout(500); // needs this else card number isn't guaranteed to fill correctly !?
+
+        await typeIntoSecuredField(creditCard, CARD_IFRAME_TITLE, CARD_IFRAME_LABEL, REGULAR_TEST_CARD);
+        await typeIntoSecuredField(creditCard, CVC_IFRAME_TITLE, CVC_IFRAME_LABEL, TEST_CVC_VALUE);
+        await typeIntoSecuredField(creditCard, EXPIRY_DATE_IFRAME_TITLE, EXPIRY_DATE_IFRAME_LABEL, TEST_DATE_VALUE);
```

Comment:
Do we need this helper function? 

The `card` model has the public `typeCardNumber`  and other methods that are aimed to be used to type in the Component secured fields. Seems like we are adding another way of doing it here.